python/Ganga/GPIDev/Lib/File/MassStorageFile.py

```diff
@@ -101,7 +101,8 @@ def _setNamePath(self, _namePattern='', _localDir=''):
     def _on_attribute__set__(self, obj_type, attrib_name):
         # This is defining the object as uncopyable from outputfiles... do we want this mechanism still?
         r = copy.deepcopy(self)
-        if _getName(obj_type) == 'Job' and attrib_name == 'outputfiles':
+        from Ganga.GPIDev.Lib.Job.Job import Job
```

Comment:
But it does. I __really__ don't want to go trapsing round fixing the imports with regard to the IGangaFile system. Everything gets pulled in from importing `Ganga.GPIDev.Lib.File` I think still and the work to fix this is better done in another PR as it's a much wider scope than this one.